nautobot_ssot/integrations/ipfabric/diffsync/diffsync_models.py

```diff
@@ -129,7 +129,7 @@ def update(self, attrs):
             device_tags = location.tags.filter(pk=safe_delete_tag.pk)
             if device_tags.exists():
                 location.tags.remove(safe_delete_tag)
-        tonb_nbutils.tag_object(nautobot_object=location, custom_field="ssot-synced-from-ipfabric")
+        tonb_nbutils.tag_object(nautobot_object=location, custom_field="ssot_last_synchronized")
```

Comment:
Seeing how many places this is needed, it might be appropriate to define a constant for `ssot_last_synchronized` somewhere?

Reply:
I'm not sure I follow. This is meant to be a date record of the last time the particular object was updated from the System of Record.

Reply:
I meant a constant like `LAST_SYNCHRONIZED_CF_NAME = "ssot_last_synchronized"` and use that rather than string literals all over the place.

Reply:
Got it, can do.